src/NHC2.spec.ts

```diff
@@ -144,5 +151,42 @@ describe('NHC2', () => {
         fakeMqttServer.server.publish(buildEvent(STATUS_CHANGED_EVENT), noop);
       });
     });
+
+    describe('trigger basic state', () => {
+      it('should emit the basic state trigger event', done => {
+        nhc2.getEvents().subscribe(event => {
+          expect(event).toStrictEqual({
+            Method: Method.DEVICES_STATUS,
+            Params: [
+              {
+                Devices: [
+                  {
+                    Properties: [{ BasicState: 'On' }],
+                    Uuid: '25ee33e3-5b9c-4171-8ede-7e94f1cb6b33',
+                  },
+                ],
+              },
+            ],
+          });
+          done();
+        });
+
+        fakeMqttServer.server.publish(buildEvent(TRIGGER_BASIC_STATE_EVENT), noop);
+      });
+
+      it('should send the basic state change command for device', async done => {
+        fakeMqttServer.server.on('published', function(packet, client) {
+          if (packet.topic === STATUS_CHANGE_COMMAND_TOPIC) {
+            expect(packet.payload.toString()).toBe(
```

Comment:
Why do we need to assert on the `toString` method? Can't we use the `toStrictEqual` as in line #158?